### PR TITLE
Displaying EVR Rewarding Details

### PR DESCRIPTION
**Pull request by Udith-Gayan**
- Added Reward Configrations
- Added Reward Info
- Added Host Count
- Limited the decimal places in total EVRs to 3.

### CODE REVIEW

File: src/pages/Registry/index.js [hunk 1]
```diff
@@ -57,15 +63,77 @@ export default function Registry() {
           key: 'Moment Size',
           value: <Tooltip title="No. of XRP ledgers per moment"><span>{config.momentSize}</span></Tooltip>
         },
+        {
+          key: 'Host Count',
+          value: <Tooltip title="No. of XRP ledgers per moment"><span>{config.hostCount}</span></Tooltip>
```
Comment (chalith) [inline src/pages/Registry/index.js:68]: Total number of registered hosts

File: src/pages/Registry/index.js [hunk 2]
```diff
@@ -57,15 +63,77 @@ export default function Registry() {
...
+            width: '37%'
+          }
+        },
+        {
+          key: 'Epoch Reward Amount',
+          value: <Tooltip title="Total amount of EVRs rewarded in one epoch"><span>{config.rewardConfiguaration.epochRewardAmount}</span></Tooltip>
+        },
+        {
+          key: 'First Epoch Reward Quota',
+          value: <Tooltip title="EVRs rewarded in one moment in first epoch"><span>{config.rewardConfiguaration.firstEpochRewardQuota}</span></Tooltip>
```
Comment (chalith) [inline src/pages/Registry/index.js:95]: EVRs rewarded per moment within the first epoch

File: src/pages/Registry/index.js [hunk 3]
```diff
@@ -57,15 +63,77 @@ export default function Registry() {
...
+
+          }
+        },
+        {
+          key: 'Current Epoch',
+          value: <Tooltip title="Current on-going epoch"><span>{config.rewardInfo.epoch}</span></Tooltip>
+        },
+        {
+          key: 'Epoch Pool',
+          value: <Tooltip title="Available amount of EVRs in the current epoch pool"><span>{+(+config.rewardInfo.epochPool).toFixed(3)}</span></Tooltip>
```
Comment (chalith) [inline src/pages/Registry/index.js:118]: epoch's reward pool

File: src/pages/Registry/index.js [hunk 4]
```diff
@@ -57,15 +63,73 @@ export default function Registry() {
...
+          key: 'Current Epoch',
+          value: <Tooltip title="Current on-going epoch"><span>{config.rewardInfo.epoch}</span></Tooltip>
+        },
+        {
+          key: 'Epoch Pool',
+          value: <Tooltip title="Available amount of EVRs in the current epoch's reward pool"><span>{+(+config.rewardInfo.epochPool).toFixed(3)}</span></Tooltip>
+        },
+        {
+          key: `Active Host Count of the Moment ${config.rewardInfo.savedMoment}`,
+          value: <Tooltip title={`No. of Active hosts in the current moment ${config.rewardInfo.savedMoment}`}><span>{config.rewardInfo.curMomentActiveHostCount}</span></Tooltip>
```
Comment (chalith) [inline src/pages/Registry/index.js:122]: Let's remove current

**Review by chalith — APPROVED** [review]

**Review by kasunkavinda — APPROVED** [review]

**Review by kithminisg — APPROVED** [review]

**Pull request merged**